nimsuggest/tests/twithin_macro_prefix.nim

```diff
@@ -202,9 +205,19 @@ echo r.age_human_yrs()
 echo r
 
 discard """
-disabled:true
 $nimsuggest --tester $file
 >sug $1
-sug;;skField;;age;;int;;$file;;167;;6;;"";;100;;Prefix
-sug;;skMethod;;twithin_macro_prefix.age_human_yrs;;proc (self: Animal): int;;$file;;169;;9;;"";;100;;Prefix
+sug;;skField;;age;;int;;$file;;170;;6;;"";;100;;Prefix
+sug;;skMethod;;twithin_macro_prefix.age_human_yrs;;proc (self: Animal): int;;$file;;172;;9;;"";;100;;Prefix
 """
+
+# Open Questions:
```

Comment:
put in block comment:
```nim
#[
Open questions:
... 
]#
```

Reply:
Done, intending to answer all these and remove all the open questions before I'm done.